src/components/overview/ClinicalSummary.js

```diff
@@ -97,182 +89,107 @@ class ClinicalSummary extends Component {
-                            data={phenotypicFeatureLabels}
-                            chartWidthHeight={this.state.chartWidthHeight}
-                            fieldLabel={"[dataset item].phenotypic_features.[item].type.label"}
-                            setAutoQueryPageTransition={this.props.setAutoQueryPageTransition}
-                        />
-                    </Spin>
-                </Col>
+                <Row style={{display: "flex", flexWrap: "wrap"}} >
+            <Col style={{ textAlign: "center" }}>
+              <Spin spinning={isFetching}>
+                <CustomPieChart
+                  title="Individuals"
+                  data={sexLabels}
+                  chartHeight={this.state.chartHeight}
+                  chartAspectRatio={this.state.chartAspectRatio}
+                  fieldLabel={"[dataset item].subject.sex"}
+                  setAutoQueryPageTransition={this.props.setAutoQueryPageTransition}
+                  autoQueryDataType={autoQueryDataType}
+                />
+              </Spin>
+            </Col>
+            <Col  style={{ textAlign: "center" }}>
+              <Spin spinning={isFetching}>
+                <CustomPieChart
+                  title="Diseases"
+                  data={diseaseLabels}
+                  chartHeight={this.state.chartHeight}
+                  chartAspectRatio={this.state.chartAspectRatio}
+                  fieldLabel={"[dataset item].diseases.[item].term.label"}
+                  setAutoQueryPageTransition={this.props.setAutoQueryPageTransition}
+                  autoQueryDataType={autoQueryDataType}
+                />
+              </Spin>
+            </Col>
+            <Col  style={{ textAlign: "center" }}>
+              <Spin spinning={isFetching}>
+                <Histogram
+                  title="Ages"
+                  data={binnedParticipantAges}
+                  chartAspectRatio={this.state.chartAspectRatio}
+                  chartHeight={this.state.chartHeight}
+                />
+              </Spin>
+            </Col>
+            <Col style={{ textAlign: "center" }}>
+              <Spin spinning={isFetching}>
+                <CustomPieChart
+                  title="Biosamples"
+                  data={biosampleLabels}
+                  chartHeight={this.state.chartHeight}
+                  chartAspectRatio={this.state.chartAspectRatio}
+                  fieldLabel={"[dataset item].biosamples.[item].sampled_tissue.label"}
+                  setAutoQueryPageTransition={this.props.setAutoQueryPageTransition}
+                  autoQueryDataType={autoQueryDataType}
+                />
+              </Spin>
+            </Col>
+            <Col  style={{ textAlign: "center" }}>
+              <Spin spinning={isFetching}>
+                <CustomPieChart
+                  title="Phenotypic Features"
+                  data={phenotypicFeatureLabels}
+                  chartHeight={this.state.chartHeight}
```

Comment:
this stuff is a different indent size than the rest of the code - should we make the linter enforce 2 spaces (or 4) for the JSX? either way is fine to me, but it would be nice to have it consistent.

Reply:
I'm on four spaces, although probably careless about pasting code in and out of classes. It should definitely be enforced! The linter already groans about indent issues elsewhere.